src/pages/studyView/addChartButton/geneLevelSelection/GeneLevelSelection.tsx

```diff
@@ -184,17 +275,36 @@ export default class GeneLevelSelection extends React.Component<
                                 options={this.molecularProfileOptions}
                                 isClearable={false}
                                 isSearchable={false}
+                                style={{ width: '100%' }}
                             />
                         </div>
                         <button
                             disabled={this.isQueryInvalid || this.hasOQL}
                             className="btn btn-primary btn-sm"
                             data-test="GeneLevelSelectionSubmitButton"
                             onClick={this.onAddChart}
+                            style={{ width: '25%' }}
                         >
                             {this.props.submitButtonText}
                         </button>
                     </div>
+                    <div style={{ width: '70%', marginTop: '5px' }}>
```

Comment:
should this div not be inside the condition, since it's empty otherwise?

Reply:
that makes total sense